lib/Echidna/Campaign.hs

```diff
@@ -42,6 +42,8 @@ import Echidna.Solidity
 import Echidna.Test
 import Echidna.Transaction
 import Echidna.Types.Campaign
+import Echidna.Types.Tx (TxCall(SolCall), Tx(Tx), TxConf, getResult, src, call, _SolCall)
```

Comment:
I think it is better to import all the constructors here..

Reply:
anything other than this? i can totally push fixes for this soon tm

Reply:
The rest looks fine to me.